client/src/components/reservation-pieces/peoplePerRes.jsx

```diff
@@ -1,11 +1,79 @@
 import React from 'react';
+import styled from 'styled-components';
+import icon from '../../../../icon.png';
+import Down from '../downArrow.jsx';
+
+const StyledPeoplePer = styled.h4`
+  width: 140px;
+  height: 30px;
+  border: 1px solid black;
+  border-radius: 10px;
+  margin: 2px 4px 2px 1px;
+`;
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const SelectWrapper = styled.select`
+  -webkit-appearance: none;
+  display: inline-block;
+  padding: 8px 45px;
+  z-index: 2;
+  background-color: rgba(0, 0, 0, 0);
```

Comment:
Perhaps consider a z-index layering system. I normally see people online use increments of 1000 for their z-index. I wouldn't arbitrarily use 2 as my z-index.